app/models/user.rb

```diff
@@ -0,0 +1,16 @@
+class User < ApplicationRecord
+  before_save :downcase_email
+
+  validates :name, presence: true, length: {maximum: Settings.user.name_max}
+  validates :email, presence: true, length: {maximum: Settings.user.email_max},
+    format: {with: Settings.user.email_regex},
+    uniqueness: {case_sensitive: false}
```

Comment:
uniqueness: {case_sensitive: false}, có tác dụng gì em

Reply:
để tắt phân biệt giữa in hoa với ko in hoa